installing/installing-preparing.adoc

```diff
@@ -128,7 +128,7 @@ Not all installation options are supported for all platforms, as shown in the fo
 //This table is for all flavors of OpenShift, except OKD. A separate table is required because OKD does not support multiple AWS architecture types. Trying to maintain one table using conditions, while convenient, is very fragile and prone to publishing errors.
 ifndef::openshift-origin[]
 |===
-||Alibaba |AWS (x86_64) |AWS (arm64) |Azure (x86_64) |Azure (arm64)|Azure Stack Hub |GCP |Nutanix |{rh-openstack} |RHV |Bare metal (x86_64) |Bare metal (arm64) |vSphere |VMC |IBM Cloud VPC |{ibmzProductName} |{ibmpowerProductName}
+||Alibaba |AWS(64-bit x86) |AWS(64-bit ARM) |Azure(64-bit x86) |Azure(64-bit ARM)|Azure Stack Hub |GCP |Nutanix |{rh-openstack} |RHV |Bare metal(64-bit x86) |Bare metal(64-bit ARM) |vSphere |VMC |IBM Cloud VPC |{ibmzProductName} |{ibmpowerProductName}
```

Comment:
Did you intend to omit a space between the platform name and the opening parens?

Reply:
Ooo, great catch! There is suppose to be a space between the platform name and the opening parenth. Will fix that! Thank you!